app/views/layouts/_header.html.erb

```diff
@@ -44,6 +44,9 @@
       <% unless @community.is_fake %>
         <%= link_to help_center_path, class: 'header--item' do %>
           <i class="fas fa-fw fa-question-circle"></i> Help
+        <% end %>
+		<%= link_to dashboard_path, class: 'header--item' do %>
```

Comment:
Odd indentation on this line. Are you using hard tabs? Most editors have an option to use soft tabs (i.e. convert tabs to spaces when you hit tab, which is what we use)

Reply:
Is now fixed